### PR TITLE
NXP-24536: add RHEL/CentOS support

### PR DESCRIPTION
**Pull request by ffjdm**

### CODE REVIEW

**Comment by tmartins** [comment]
:+1:

File: nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh [hunk 1]
```diff
@@ -27,13 +27,15 @@ readonly TOMCAT_LATEST_URL="http://www.apache.org/dist/tomcat/tomcat-7"
...
   echo
   echo "Note: If no target version is specified the latest one will be retrieved from the Tomcat site"
   echo
+  echo "Warning: this script cannot be run with sudo -c. The Nuxeo user has to be logged in."
+  echo
   echo "Examples:"
-  echo -e "\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat"
-  echo -e "\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat 7.0.76"
+  echo -e "\\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat"
+  echo -e "\\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat 7.0.76"
```
Comment (efge) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh]: This is a cleanup and should be a separate commit from the one fixing the basic RHEL/CentOS problem.

File: nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh [hunk 2]
```diff
@@ -43,11 +45,11 @@ verifyHash() {
...
   local hash_value
 
-  echo -n -e "\tVerifying ${hash_filename}..."
+  echo -n -e "\\tVerifying ${hash_filename}..."
 
   # file containing the hash
-  filename_tocheck="$(cat "${hash_filename}" | cut -d ' ' -f 2 | sed 's/^[\s*]//;s/\s*$//')" || { echo -e "FAILED"; return 1; }
-  hash_tocheck="$(cat "${hash_filename}" | cut -d ' ' -f 1 | sed 's/^[\s*]//;s/\s*$//')" || { echo -e "FAILED"; return 1; }
+  filename_tocheck="$(cut -d ' ' -f 2 < "${hash_filename}" | sed 's/^[\s*]//;s/\s*$//')" || { echo -e "FAILED"; return 1; }
+  hash_tocheck="$(cut -d ' ' -f 1 < "${hash_filename}" | sed 's/^[\s*]//;s/\s*$//')" || { echo -e "FAILED"; return 1; }
```
Comment (efge) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh]: Cleanup —> separate commit.

File: nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh [hunk 3]
```diff
@@ -72,7 +74,7 @@ if [ ! -f "$1/templates/nuxeo.defaults" ]; then
   echo
   exit 1
 fi
-NUXEO_HOME=$1
+NUXEO_HOME="$( cd "${1}" && pwd )"
```
Comment (efge) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh]: Cleanup? Unclear why this is changed.

File: nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh [hunk 4]
```diff
@@ -152,23 +149,23 @@ rm "${NUXEO_HOME}/nxserver/lib/tomcat-juli-${TOMCAT_SOURCE}.jar"
...
-echo -e "\tUpdating Tomcat version in nuxeo.defaults..."
+echo -e "\\tUpdating Tomcat version in nuxeo.defaults..."
 sed 's/'"${TOMCAT_SOURCE//./\\.}"'/'"${TOMCAT_TARGET}"'/g' "${NUXEO_HOME}/templates/nuxeo.defaults" > "${WORK_FOLDER}/nuxeo.defaults"
 mv "${WORK_FOLDER}/nuxeo.defaults" "${NUXEO_HOME}/templates/nuxeo.defaults"
 
 echo
 echo "Deleting temporary folder..."
 echo
-rm -rf "${WORK_FOLDER}"
+rm -rf "${WORK_FOLDER}"
```
Comment (efge) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh]: You should keep a newline at the end of the file.
